.github/workflows/dependabot-auto-merge-go.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     steps:
```

Comment:
Should we also add the [release action](https://github.com/ConduitIO/conduit-connector-file/blob/main/.github/workflows/release.yml)?